### ISSUE TITLE
Insufficient Color Contrast

### ISSUE THREAD

**Issue by JacquelineOmollo** [unitymakesus/justgetsimple#66]
- [x] Homepage H1
- [x] 2:00pm, recurring every other Monday
- [x] LEARN MORE
- [x] Fragile X Syndrome FXS

**Comment by andrewmarino** [comment]
Fixed instances of insufficient color contrast on the homepage. Will reopen if other instances are found on interior pages.
